app/src/common/shared/org/mozilla/vrbrowser/browser/engine/Session.java

```diff
@@ -767,6 +767,9 @@ public void goForward() {
     }
 
     public void setActive(boolean aActive) {
+        if (mState.mSession != null && mState.isActive() == aActive) {
+            return;
+        }
```

Comment:
I think `setActive(true);` is okay and since we can get out of sync, I think we should just guard against `setActive(false)` twice?